examples/common.h

```diff
@@ -78,6 +78,9 @@ struct gpt_params {
     bool mem_test          = false; // compute maximum memory usage
     bool export_cgraph     = false; // export the computation graph
     bool verbose_prompt    = false; // print prompt tokens before generation
+
+    int bos_token  = 1; // beginning of sentence token
```

Comment:
Relating to style.
Can we replace with an enum
```
enum token_pos { bos_token, eos_token };
```
Or replace with other options